omas/machine_mappings/d3d.py

```diff
@@ -1504,32 +1513,35 @@ def core_profiles_profile_1d(ods, pulse, PROFILES_tree="OMFIT_PROFS", PROFILES_r
         pulse_id = pulse
         if PROFILES_run_id is not None:
             pulse_id = int(str(pulse) + PROFILES_run_id)
-        omfit_profiles_node = '\\TOP.'
-        query = {
-            "electrons.density_thermal": "N_E",
-            "electrons.density_fit.measured": "RW_N_E",
-            "electrons.temperature": "T_E",
-            "electrons.temperature_fit.measured": "RW_T_E",
-            "ion[0].density_thermal": "N_D",
-            "ion[0].temperature": "T_D",
-            "ion[1].velocity.toroidal": "V_TOR_C",
-            "ion[1].velocity.toroidal_fit.measured": "RW_V_TOR_C",
-            "ion[1].density_thermal": "N_C",
-            "ion[1].density_fit.measured": "RW_N_C",
-            "ion[1].temperature": "T_C",
-            "ion[1].temperature_fit.measured": "RW_T_C",
-        }
+        query = OrderedDict()
```

Comment:
Order of setting in the ods is quite important. The axis for each quantity must be set first, otherwise OMAS complains.